### DISCUSSION TITLE
Created Presigned URL for Multpart Upload

### DISCUSSION THREAD

**Discussion by zhoub** [aws/aws-sdk-go-v2#1654] [Q&A]
Confirm by changing [ ] to [x] below:
- [x ] I've gone though the [API reference](https://github.com/aws/aws-sdk-go-v2/issues)
- [x ] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-go) for answers

**Describe the question**

Wanted to create presigned URL for each part when multipart uploading is being used to upload big file from external.

As Python example
https://github.com/T-Tmnr/aws_s3_multipartupload_with_presigned_url/blob/927685c378c4e929ce606dbec735304a3e49a0bd/aws-multipart-upload.py#L38
```python
 signed_url = self.s3.generate_presigned_url(
            ClientMethod='upload_part',
            Params={'Bucket': self.bucket_name,
                    'Key': self.key,
                    'UploadId': self.upload_id,
                    'PartNumber': part_no},
            ExpiresIn=expire)
```
It's able to specify **UploadId** and **PartNumber**.

Is that possible to do so in Golang to generate Presigned URL for each part ?

Thank you in advance !

**Comment by jerisalan** [comment]
Its already available via the API linked here: https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/s3#PresignClient.PresignUploadPart
